tools/data/activitynet/download.py

```diff
@@ -0,0 +1,124 @@
+# This scripts is copied from
+# https://github.com/activitynet/ActivityNet/blob/master/Crawler/Kinetics/download.py  # noqa: E501
```

Comment:
mention its licence

Reply:
It is MIT license. How to properly add it to the head of code? I browsed some repos and didn't find the answer.

Reply:
add a line of comment like
```
The code is licensed under the MIT licence.
```